Doc/c-api/long.rst

```diff
@@ -89,9 +89,11 @@ All integers are implemented as "long" integer objects of arbitrary size.
    determined based on the leading characters of *str*: if *str* starts with
    ``'0x'`` or ``'0X'``, radix 16 will be used; if *str* starts with ``'0o'`` or
    ``'0O'``, radix 8 will be used; if *str* starts with ``'0b'`` or ``'0B'``,
-   radix 2 will be used; otherwise radix 10 will be used.  If *base* is not
-   ``0``, it must be between ``2`` and ``36``, inclusive.  Leading spaces are
-   ignored.  If there are no digits, :exc:`ValueError` will be raised.
+   radix 2 will be used; if *str* starts with any number other than ``0``, then
+   radix 10 will be used; otherwise :exc:`ValueError` will be raised.  If *base* is not
```

Comment:
I don’t think forcing the ValueError case to the end is that important, and I though the text “if . . . [the prefix is] other than 0 . . .; otherwise” is harder to understand than it could be. But this is only a minor thing though; if you think putting ValueError at the end is important, so be it :)

Bullet points might simplify it; that did cross my mind. Another option would be to drop the details and just link to the Lexical Analysis section for the details. That would eliminate the problem with a string like "000" not “starting with a number other than 0”, yet presumably this doesn’t raise ValueError.